MovingBlocks/Terasology
# Fix Groovy Wrapper Logger Warning

### Description

Groovy Wrapper currently gives a logger warning whenever it tries to run a git command internally. This PR fixes the warning by importing the sl4fj logger class into the common groovy script.

### How to test

Try running `groovyw module get <name>` with and without the fix. The warning should be gone after the fix. 

_Note_ - Make sure to delete the module before re-using the command because the script will skip the git commands and the fix will have no effect in that situation.
The warning is only shown when the groovy script uses the git module.

### Result

![1](https://user-images.githubusercontent.com/16421831/55495429-9cef9a00-5688-11e9-82ad-d4e69ce8c663.PNG)
_**Before fixing**_

![2](https://user-images.githubusercontent.com/16421831/55495509-c6102a80-5688-11e9-9e74-fce1d3c612b4.PNG)
**_After fixing_**
